src/formats/PDB.cpp

```diff
@@ -81,13 +94,25 @@ void PDBFormat::read_step(const size_t step, Frame& frame) {
 void PDBFormat::read(Frame& frame) {
     frame.resize(0);
     residues_.clear();
+    atom_offsets_.clear();
 
     std::streampos position;
     bool got_end = false;
     while (!got_end && !file_->eof()) {
         auto line = file_->readline();
         auto record = get_record(line);
         switch (record) {
+        case Record::HEADER:
+            if (line.size() < 66) {continue;}
+            frame.set("classification", trim(line.substr(10, 40)));
+            frame.set("deposition_date", trim(line.substr(50, 9)));
+            frame.set("pdb_idcode", trim(line.substr(62, 4)));
+            continue;
+        case Record::TITLE:
+            frame.set("name", trim(
+                      frame.get<Property::STRING>("name").value_or("") +
+                      line.substr(10, 70)));
```

Comment:
Should we add a check on the line length here too?

Reply:
Sure